gcp/api/integration_tests.py

```diff
@@ -49,50 +49,193 @@ class IntegrationTests(unittest.TestCase,
   """Server integration tests."""
 
   _VULN_890 = {
-      'published': '2023-09-21T14:01:03.576514Z',
-      'schema_version': '1.6.0',
-      'affected': [{
-          'database_specific': {
-              'source': 'https://github.com/google/oss-fuzz-vulns/'
-                        'blob/main/vulns/libdwarf/OSV-2023-890.yaml'
-          },
-          'ecosystem_specific': {
-              'severity': 'HIGH'
-          },
-          'package': {
-              'ecosystem': 'OSS-Fuzz',
-              'name': 'libdwarf',
-              'purl': 'pkg:generic/libdwarf'
-          },
-          'ranges': [{
-              'events': [{
-                  'introduced': 'b55ce0185528bf0a99e375cf8f3c84b76b6881a3'
-              }, {
-                  'fixed': 'cd741379bd0203a0875b413542d5f982606ae637'
+      'vulns': [{
```

Comment:
Let's save CVE-2024-2002 in a separate variable, and just mention both of them in the assert.